ui/selectors/multichain-accounts/account-tree.ts

```diff
@@ -688,3 +688,36 @@ export const getNetworkAddressCount = createSelector(
     return wallet.groups[accountGroupId].accounts.length;
   },
 );
+
+/**
+ * Get all internal accounts from a specific account group by its ID.
+ *
+ * @param state - The Redux state.
+ * @param groupId - The ID of the account group.
+ * @returns Array of internal accounts in the specified group, or empty array if not found.
+ */
+export const getInternalAccountsFromGroupById = createDeepEqualSelector(
```

Comment:
Need to filter out test networks and add tests

Reply:
We are filtering out the test networks at the mutlichain-account-address-row level https://github.com/MetaMask/metamask-extension/pull/35191/files#r2299630842